### PR TITLE
[NSA-8355] Fix blacklisted email validation on invite

### PR DESCRIPTION
**Pull request by insacuri**

### CODE REVIEW

File: test/unit/app/users/postNewUserDetails.test.js [hunk 1]
```diff
@@ -173,9 +173,9 @@ describe('when entering a new users details', () => {
     });
   });
 
-  it('then it should render view if email is a blacklisted email and environment is Production', async () => {
+  it('should render with error if emailValidation is undefined and the email address is a blacklisted email', async () => {
     req.body.email = 'blacklisted.domain@hotmail.com';
-    config.toggles.environmentName = 'pr';
+    process.env.emailValidation = 'true';
```
Comment (JaredAMarsh) [inline test/unit/app/users/postNewUserDetails.test.js:178]: For this test, emailValidation should be undefined I believe? 😄 Easy thing to miss!

**Review by JaredAMarsh — CHANGES_REQUESTED** [review]
I completely agree with the implementation of opt-out of validation rather than opt-in, and it'll help to catch issues with the pipelines in specific environments 👍 

Only quick change (I may have misread) is one of the tests where `emailValidation` should be `undefined` but is set to `true` 👍

File: test/unit/app/users/postNewUserDetails.test.js (hunk 1 above)
Comment (insacuri) [reply test/unit/app/users/postNewUserDetails.test.js]: Fixed!

**Review by JaredAMarsh — APPROVED** [review]
Unit test correct, and matches implementation 👍

**Review by shravani-kondaparti — APPROVED** [review]

**Review by shravani-kondaparti — APPROVED** [review]

**Pull request merged**